guide/phone_store/test/phone_store_service_test.bal

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2018 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/test;
+
+@test:Config
+function testResourceGetPhoneList() {
+
+    endpoint http:Client httpEndpoint { url:"http://localhost:9090/phonestore" };
+    // Initialize the empty http request
+    http:Request req;
+
+    // Send a 'get' request and obtain the response
+    http:Response response = check httpEndpoint->get("/getPhoneList", message = req);
```

Comment:
We don't need to initialize empty request here. We can call like,
http:Response response = check httpEndpoint->get("/getPhoneList");
Please check